supply/lib/supply/options.rb

```diff
@@ -164,9 +165,17 @@ def self.available_options
                                        value.each do |path|
                                          UI.user_error! "Could not find mapping file at path '#{path}'" unless File.exist?(path)
                                        end
+                                     end),
+        FastlaneCore::ConfigItem.new(key: :root_url,
+                                     env_name: "SUPPLY_ROOT_URL",
+                                     description: "Root URL for the Google Play API",
```

Comment:
Will do. Would it be clearer if I called it `base URL`? I used `root` because that's what Google's client calls it, but happy to change if it is clearer.